### PR TITLE
Fixed the issues as told

### PR DESCRIPTION
**Pull request by Sirsho29**
The subscribe button is shifted to extreme right.

All Spin cubes are having padding of 8.0 in all sides.

The View All Societies button is having white text.

### CODE REVIEW

File: lib/splash_screen.dart [hunk 1]
```diff
@@ -43,9 +43,15 @@ class SplashScreen extends StatelessWidget {
                 alignment: AlignmentDirectional.topStart // or Alignment.topLeft
                 ),
             SizedBox(height: 30),
-            SpinKitFadingCube(
-              color: Colors.white,
-              size: 50.0,
+            Padding(
```
Comment (mannprerak2) [inline lib/splash_screen.dart:46]: Should only be 1 instead of 2 padding widgets here.

File: lib/ui/bottom_nav_bar.dart [hunk 2]
```diff
@@ -1,3 +1,5 @@
+import 'dart:html';
```
Comment (mannprerak2) [inline lib/ui/bottom_nav_bar.dart]: Remove this import.

File: lib/ui/tabs/subs_tab.dart [hunk 3]
```diff
@@ -1,3 +1,5 @@
+import 'dart:ui';
+
```
Comment (mannprerak2) [inline lib/ui/tabs/subs_tab.dart]: Unused import?

File: lib/ui/tiles/society_tile_left.dart [hunk 4]
```diff
@@ -57,12 +57,14 @@ class SocietyTileLeft extends StatelessWidget {
               ),
               Expanded(
                 child: Padding(
-                  padding: const EdgeInsets.all(4.0),
+                  padding: const EdgeInsets.all(0.0),
```
Comment (mannprerak2) [inline lib/ui/tiles/society_tile_left.dart]: Padding `0.0` makes no sense, either remove this or add padding.

File: lib/ui/tiles/subscribe_button.dart [hunk 5]
```diff
@@ -38,12 +38,12 @@ class _SubscribeButtonState extends State<SubscribeButton> {
 
   @override
   Widget build(BuildContext context) {
-    return IconButton(
```
Comment (mannprerak2) [inline lib/ui/tiles/subscribe_button.dart:41]: Why is this change needed?

**Review by mannprerak2 — CHANGES_REQUESTED** [review]
Hi @Sirsho29 please check my inline comments 😄

File: lib/ui/tiles/society_tile_left.dart [hunk 6]
```diff
@@ -57,12 +57,14 @@ class SocietyTileLeft extends StatelessWidget {
...
                   child: Column(
                     crossAxisAlignment: CrossAxisAlignment.start,
                     children: <Widget>[
 
-                      FittedBox(
+                      Container(
+                        width: MediaQuery.of(context).size.width,
+
```
Comment (mannprerak2) [inline lib/ui/tiles/society_tile_left.dart]: `width: MediaQuery.of(context).size.width` is probably not needed because the Row will automatically cover all the given space.
So you can remove the Container Widget completely.

File: lib/ui/tiles/subscribe_button.dart (hunk 5 above)
Comment (Sirsho29) [reply lib/ui/tiles/subscribe_button.dart:41]: Because Icon Button takes a liitle space apart from the icon and thats why the Subscribe button is not shifted to the extreme right. Thats why I changed to Gesture detector.

Comment (mannprerak2) [reply lib/ui/tiles/subscribe_button.dart:41]: Alright.

File: lib/ui/tiles/society_tile_left.dart (hunk 6 above)
Comment (Sirsho29) [reply lib/ui/tiles/society_tile_left.dart]: Fixed

File: lib/ui/tiles/society_tile_left.dart (hunk 4 above)
Comment (Sirsho29) [reply lib/ui/tiles/society_tile_left.dart]: Fixed

File: lib/splash_screen.dart (hunk 1 above)
Comment (Sirsho29) [reply lib/splash_screen.dart:46]: Fixed

File: lib/ui/bottom_nav_bar.dart (hunk 2 above)
Comment (Sirsho29) [reply lib/ui/bottom_nav_bar.dart]: Fixed

File: lib/ui/tabs/subs_tab.dart (hunk 3 above)
Comment (Sirsho29) [reply lib/ui/tabs/subs_tab.dart]: Fixed

File: lib/ui/tiles/society_tile_left.dart (hunk 4 above)
Comment (Sirsho29) [reply lib/ui/tiles/society_tile_left.dart]: Fixed

File: lib/ui/tiles/society_tile_left.dart (hunk 6 above)
Comment (Sirsho29) [reply lib/ui/tiles/society_tile_left.dart]: Fixed
